cablelabs/snaps-kubernetes
# Ha ci scripts

#### What does this PR do?
Fixes #118 
Adds CI scripts to test HA K8s
Refactoring some duplicate helper functions out.
Removed a few unused variables from the load balancer setup playbooks
#### Do you have any concerns with this PR?
no
#### How can the reviewer verify this PR?
run CI and ensure that one can issue kubectl calls from the build node and the generated node-kubeconfig.yaml file. Also ensure that there are multiple master nodes running
#### Any background context you want to provide?
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
yes
- Does the documentation need an update?
not really with this PR, but it should be looked at to ensure HA configuration is documented properly
- Does this add new Python dependencies?
No
- Have you added unit or functional tests for this PR?
This was the main goal of this exercise
- Does this patch update any configuration files?
There are some sample configuration files that were changed for testing purposes, but this PR has not caused any changes to the config structure.
